src/style.css

```diff
@@ -1 +1,8 @@
-/* styles go here */
```

Comment:
Consider adding more detailed comments to describe the purpose of each CSS rule or group of rules, especially for those that may not be self-explanatory. This can help improve code readability and maintainability. 📚